index.js

```diff
@@ -9,8 +10,17 @@ var isGzipped = function(data) {
   return true
 }
 
-module.exports = function() {
+var createParser = function() {
+  var zlibGunzip = zlib.createGunzip()
+  var gunzipObj = gunzip()
+  zlibGunzip.pipe(gunzipObj)
+  return duplexify(zlibGunzip, gunzipObj)
+}
+
```

Comment:
we can just use [pumpify](https://github.com/mafintosh/pumpify) for this :)

``` js
var createParser () {
  return pumpify(zlib.createGunzip(), gunzip())
}
```

or even inline it since its just a single line now :)